account-kit/infra/src/gas-manager.ts

```diff
@@ -83,13 +99,24 @@ export const PermitTypes = {
   ],
 } as const;
 
-export const EIP712NoncesAbi = [
+export const EIP20Abis = [
```

Comment:
This is not an ERC-20 ABI, not all ERC-20 tokens have a `nonces` function. This should be named `ERC7597Abis`, which applies to ERC-20 tokens with 2612 extended to smart accounts.